README.md

```diff
@@ -166,6 +168,17 @@ simply `npm install` and `npm test`.
     provisions of this method, which you cannot possibly
     obtain in legacy engines.
 
+### Example of applying ES compatability shims in a browser project
+
+```html
+<script src="es5-shim.js"></script>
+<script src="es5-sham.js"></script>
+<script src="json3.js"></script>
```

Comment:
ah, if json3 is in fact ES5.1 with nothing extra then sure, why not.

we should probably link all of these to cdnjs so that the examples are copypasteable